src/javascript/app/pages/user/new_account/financial_acc_opening.js

```diff
@@ -55,6 +64,16 @@ const FinancialAccOpening = (() => {
         });
 
         Promise.all([req_settings, req_financial_assessment]).then(() => {
+            const client_form_response = sessionStorage.getItem('client_form_response') ? JSON.parse(sessionStorage.getItem('client_form_response')).echo_req : {};
```

Comment:
let's store this `sessionStorage.getItem('client_form_response')` instead of fetching it twice

Reply:
Thanks, this is indeed a better way 😃